app/src/main/java/io/horizontalsystems/bankwallet/modules/createwallet/CreateWalletActivity.kt

```diff
@@ -0,0 +1,110 @@
+package io.horizontalsystems.bankwallet.modules.createwallet
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.RecyclerView
+import io.horizontalsystems.bankwallet.BaseActivity
+import io.horizontalsystems.bankwallet.R
+import io.horizontalsystems.bankwallet.core.EosUnsupportedException
+import io.horizontalsystems.bankwallet.core.setOnSingleClickListener
+import io.horizontalsystems.bankwallet.modules.main.MainModule
+import io.horizontalsystems.bankwallet.ui.dialogs.AlertDialogFragment
+import io.horizontalsystems.bankwallet.ui.extensions.TopMenuItem
+import kotlinx.android.extensions.LayoutContainer
+import kotlinx.android.synthetic.main.activity_create_wallet.*
+import kotlinx.android.synthetic.main.view_holder_cell_multiline.*
+
+class CreateWalletActivity : BaseActivity() {
+    private lateinit var presenter: CreateWalletPresenter
+    private lateinit var coinItemsAdapter: CoinItemsAdapter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_create_wallet)
+        shadowlessToolbar.bind(
+                title = getString(R.string.Create_Title),
+                leftBtnItem = TopMenuItem(text = R.string.Button_Cancel, onClick = { onBackPressed() }),
+                rightBtnItem = TopMenuItem(text = R.string.Button_Create, onClick = { presenter.didClickCreate() })
+        )
+
+        presenter = ViewModelProviders.of(this, CreateWalletModule.Factory()).get(CreateWalletPresenter::class.java)
+
+        observeView(presenter.view as CreateWalletView)
+        observeRouter(presenter.router as CreateWalletRouter)
+
+        coinItemsAdapter = CoinItemsAdapter(presenter)
+        coinItemsAdapter.setHasStableIds(true)
```

Comment:
adapter.setHasStableIds(true) can be added inside init{} of adapter. Since with this option enabled in adapter we need to add getItemId() too